.github/workflows/cd-frontend-infra.yml

```
@@ -0,0 +1,61 @@
+name: Deploy Frontend Infra
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "infra/frontend/**"
+
+jobs:
+  build-repository:
+    name: Build Repository
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: 1.7.1
+          terraform_wrapper: false
+
+      - name: Configure AWS credentials
+        uses: ./.github/actions/configure-aws-credentials
+        with:
+          app_name: analytics
+          environment: shared
+      - name: Build Repository
+        run: |
+          ./bin/terraform-init-and-apply.sh infra/frontend/build-repository shared
+
+  deploy-infra:
+    name: Deploy Infrastructure
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        directory: ["database", "service"]
```

```suggestion
        directory: ["service"]
```

frontend doesn't have a database

---

fixed in https://github.com/HHS/simpler-grants-gov/pull/1839